.gitignore

```diff
@@ -17,6 +17,8 @@ results.txt
 results.json.txt
 worktodo.txt
 worktodo.add
+*.ini
+!src/*.ini
```

Comment:
That could be written as `/*.ini` meaning that `*.ini` is only ignored in the same directory as `.gitignore`: https://git-scm.com/docs/gitignore